src/draft-service.js

```diff
@@ -0,0 +1,355 @@
+// @flow
+'use strict'
+
+import _differenceBy from 'lodash.differenceby'
+import { v4 as uuidv4 } from 'uuid'
+import localForage from 'localforage'
+
+import OneBlinkAppsError from './services/errors/oneBlinkAppsError'
+import { isOffline } from './offline-service'
+import { getUsername, isLoggedIn } from './services/cognito'
+import { getIssuerFromJWT } from './auth-service'
+import { uploadDraftData, putDrafts } from './services/api/drafts'
```

Comment:
```suggestion
import { uploadDraftData, putDrafts } from './services/api'
```

Reply:
I have removed these in a later pull request too. 👍  Was just more code to maintain.